src/components/Heading/_Heading.scss

```diff
@@ -89,7 +89,8 @@
   }
 }
 
-.nypl-ds {
+// .nypl-ds {
```

Comment:
This file wasn't deleted.

Reply:
Good catch.  Also had to cleanup how `Card` was handling styles for internal text/heading elements.